### DISCUSSION TITLE
Sub menus help

### DISCUSSION THREAD

**Discussion by fadytalat268** [daniele77/cli#164] [Q&A] [answered]
how to get the help info of a submenu without getting into it ? can command as following be used "> <submenu name> help" ?

**Comment by daniele77** [comment, marked as answer]
Unfortunately, it's not possible (yet).
However, if you think it's a nice-to-have feature, you can fill in a new GitHub issue.
Thanks.

**Comment by fadytalat268** [comment]
okay thanks
